src/ParallelHelper.Plugin/source.extension.vsixmanifest

```diff
@@ -11,9 +11,24 @@
         <Tags>C#; Parallel; Asynchronous; Concurrency; Bugs; Best Practices; TPL; Task; QC; Static Analysis; async; await</Tags>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.3,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.3,17.0)" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.3,17.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.3,17.0)" >
+          <ProductArchitecture>x86</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.3,17.0)" >
+          <ProductArchitecture>x86</ProductArchitecture>
+        </InstallationTarget>
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.3,17.0)" >
+          <ProductArchitecture>x86</ProductArchitecture>
+        </InstallationTarget>
+      <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0,18.0)">
```

Comment:
The indentation of the following InstallationTarget Nodes is different from the rest of the file (2 in contrast to the existing 4). The same goes for the `ProductArchitecture` nodes above.